requirements.txt

```diff
@@ -12,11 +12,11 @@ icalendar>=3.9.2
 ipaddress>=1.0.16              ; python_version < '3.3'
 Jinja2>=2.8
 markupsafe>=0.23
-matplotlib>=1.5.1
+#matplotlib>=1.5.1
 msgpack-python>=0.4.7
 paramiko>=2.0.0
 pluginbase>=0.4
-psycopg2>=2.6.1
+#psycopg2>=2.6.1
```

Comment:
same, this PR does not require any changes to the lib requirements of King Phisher.